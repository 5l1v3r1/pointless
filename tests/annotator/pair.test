
b = {x: y}
y = 1
x = "asdf"