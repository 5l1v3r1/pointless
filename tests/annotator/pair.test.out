
b = {x: y}
access: x global: 2
access: y global: 1
entry: b index: 0

y = 1
entry: y index: 1

x = "asdf"
entry: x index: 2
